apps/partners-reference/cypress/integration/navigation.ts

```diff
@@ -0,0 +1,23 @@
+describe("Navigating around the site", () => {
+  it("Loads the homepage directly", () => {
+    cy.visit("/")
+    // Check if sign in prompt is displayed
+    cy.contains("Sign In")
+  })
+
+  it("Signs in", () => {
+    cy.visit("/")
+    cy.fixture("user").then((user) => {
+      cy.get("input#email").type(user.email)
+      cy.get("input#password").type(user.password)
+      cy.get(".button.is-filled").contains("Sign In").click()
+      cy.contains("This will be the home page")
```

Comment:
Do we maybe want to check some other text on the home page that isn't likely to change as we build out the app? This basically guarantees that we'll have to update this test as soon as we modify the home page. Maybe check that the title is "Housing Portal", or something like that?